lib/questionnaires/model/src/rendering_string.dart

```diff
@@ -44,6 +48,7 @@ class RenderingString with Diagnosticable {
     this.renderingStyle,
     this.renderingXhtml,
     this.renderingMarkdown,
+    this.extensions,
```

Comment:
I'm wondering if we still need to keep the extensions as a public property of RenderingString. I see were not using renderingString.extensions anymore.

Reply:
Thanks for the reminder, removed [here](https://github.com/sujrd/faiadashu/pull/49/commits/0f82ef052054f94c403434c34b87bc37010ec48d).